templates/snap-details.html

```diff
@@ -97,14 +97,35 @@ <h1 class="p-heading--two p-snap-heading__name">{{ snap_title }}</h1>
         </table>
       </div>
     </div>
-    {% if countries %}
-      <div class="row js-snap-map-holder" style="margin-top: 2rem;">
-        <div class="col-12">
+    <div class="row" style="margin-top: 2rem;">
+      {% if countries %}
+        <div class="{% if normalized_os %}col-8{% else %}col-12{% endif %} js-snap-map-holder">
           <h4>Where people are using {{ snap_title }}</h4>
           <div id="js-snap-map" class="snapcraft-territories"></div>
         </div>
-      </div>
-    {% endif %}
+      {% endif %}
+      {% if normalized_os %}
+        <div class="col-4">
+          <h4>Users by distribution</h4>
+          <div class="snapcraft-distro-chart">
+            <div class="snapcraft-distro-chart__names">
+              {% for distro in normalized_os %}
+                <div class="snapcraft-distro-chart__name">{{ distro.name }}</div>
+              {% endfor %}
+            </div>
+            <div class="snapcraft-distro-chart__bars">
+              {% for distro in normalized_os %}
+                <div
+                  class="snapcraft-distro-chart__bar snapcraft-distro-chart--color-{{ (loop.index % 12) + 1 }}"
+                  title="{{ distro.value * 100 }}"
```

Comment:
Tooltip shows a number, but we don't explain anywhere what this number means.
I can just guess that it's kind of a percent, as the top one says '100', but it's confusing.

<img width="338" alt="screen shot 2018-04-24 at 17 30 07" src="https://user-images.githubusercontent.com/83575/39197645-512eb2f2-47e5-11e8-8d20-e6b281eaeba1.png">

We should either remove tooltip or make it clearer what the number there means. Also if we want a tooltip it should be a vanilla tooltip like on the map.